cmd/podman/common/create.go

```diff
@@ -383,8 +383,7 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 			"read-only", podmanConfig.ContainersConfDefaultsRO.Containers.ReadOnly,
 			"Make containers root filesystem read-only",
 		)
-		createFlags.BoolVar(
-			&cf.ReadWriteTmpFS,
+		createFlags.Bool(
 			"read-only-tmpfs", cf.ReadWriteTmpFS,
 			"When running containers in read-only mode mount a read-write tmpfs on /run, /tmp and /var/tmp",
```

Comment:
I realize this is not part of your PR, but this description seems absolutely wrong and misleading. It seems to apply to the variable name, not the option name. (Also, missing comma between "mode" and "mount").